sagemaker-triton/fil_ensemble/README.md

```diff
@@ -0,0 +1,19 @@
+# XGBoost model inference pipeline with NVIDIA Triton Inference Server on Amazon SageMaker
+
+In this example we show an end-to-end GPU-accelerated fraud detection example making use of tree-based models like XGBoost. In the first notebook [1_prep_rapids_train_xgb.ipynb](1_prep_rapids_train_xgb.ipynb) we demonstrate GPU-accelerated tabular data preprocessing using RAPIDS and training of XGBoost model for fraud detection on the GPU in SageMaker. Then in second notebook [2_triton_xgb_fil_ensemble.ipynb](2_triton_xgb_fil_ensemble.ipynb) we walkthrough the process of deploying data preprocessing + XGBoost model inference pipeline for high throughput, low-latency inference on Triton in SageMaker. 
```

Comment:
I suggest using the notebook title as link text, rather than the raw notebook filenames. Ie `[Pre-processing + XGBoost model inference pipeline with NVIDIA Triton Inference Server on Amazon SageMaker](1_prep_rapids_train_xgb.ipynb)`. But it's okay as-is if you prefer the raw filename.

Grammar nits: `In second notebook` -> `In the second notebook` , `we walkthrough` -> `we walk through`. Also the `+` is a bit casual, could be revised to `and`. 

It could be worth adding a sentence or phrase defining what Triton is/does, in case customers encounter this example but aren't familiar with it.

Reply:
Updated with suggested results.